Jenkinsfile

```
@@ -63,54 +100,46 @@ def buildImageAndRunTests(params, String suffix) {
             sh "mv -fv kernel-${suffix}* tarball/"
         }
     }
+}
+
+def runTests(param, String suffix) {
```

```suggestion
def runTests(params, String suffix) {
```
is I assume the origin of the "qemu-null" failure (through params.buildOS below)

---

Thanks